src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.html

```diff
@@ -1,3 +1,4 @@
+<h3 i18n>{{ title }}</h3>
```

Comment:
@pcuzner please have a look at this

Reply:
I agree with @Devp00l here. I wonder if we could remove the headline between the tab and the table?

Reply:
Yes, agree on removing the title for the tables which already have title in tab itself. But for the tables like Configuration/Manager modules, RGW users/buckets, NFS/File systems having the title doesn't make it that redundant. So, should we keep titles for these tables. Any thoughts?

RGW users/buckets table view:
![RGW user list](https://user-images.githubusercontent.com/23611106/95553088-7784de80-0a2b-11eb-8666-d6557bacab64.png)

NFS/File systems table view:
![NFS](https://user-images.githubusercontent.com/23611106/95553099-7bb0fc00-0a2b-11eb-91cb-d2ed18791c1b.png)

Configuration/Manager modules table view
![Configuration](https://user-images.githubusercontent.com/23611106/95553110-7fdd1980-0a2b-11eb-9d5e-dd19b8b7f92d.png)

Reply:
>  But for the tables like Configuration/Manager modules, RGW users/buckets, NFS/File systems having the title doesn't make it that redundant. So, should we keep titles for these tables?

As per our discussion during the standup, it would be fine to keep these. Thanks!